src/widgets/DataTable/TextInputCell.js

```diff
@@ -55,7 +56,7 @@ const TextInputCell = React.memo(
 
     const { focusNextCell, getRefIndex, getCellRef } = React.useContext(RefContext);
     const refIndex = getRefIndex(rowIndex, columnKey);
-
+    const onFocus = () => onFocus(rowKey, columnKey);
```

Comment:
hmm I think you need to get the `onFocus` function from a prop and then you can create the function here